index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+</head>
+
+<body>
+  <header id="brand" class="toolbar">
+    <div class="my-logo">
+      Richard Burneo
+    </div>
+    <nav class="menu-botton">
+      <button><i class="fa fa-bars fa-spin"></i></button>
+    </nav>
```

Comment:
- [ ] Kindly remove the `fa-spin` class from the menu bar icon, as it doesn't follow the template design. While it's good for learning animation, it doesn't fit the portfolio's aesthetics. 
- [ ] It'll be good if you could add a `padding:10px` to the header as well, to follow the template design you've picked. Currently, the spacing is not enough.
- [ ] It's also a good idea to style the menu bars to fit in with the provided template design or remove the menu bars from the button tag as they currently are taking on features of a button.

 _See images below_

**Original** | **Your project**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/164888520-a79a5067-70de-4b41-bfc8-e6bb4edcdf8d.png) | ![image](https://user-images.githubusercontent.com/49621279/164888545-07398935-33ec-4cea-ada0-ff02d30b811f.png)

Reply:
Thank you for the review.
I have finished:
- Change the menu bar icon.
- Add padding:10px  to the header:
- Edit menu bars similar to the template design.